tests/test_single_topology_stereo.py

```diff
@@ -0,0 +1,486 @@
+import functools
+
+import jax
+import numpy as np
+import scipy
+from rdkit import Chem
+
+from timemachine.fe import geometry, utils
+from timemachine.fe.single_topology import SingleTopologyV2
+from timemachine.ff import Forcefield
+from timemachine.integrator import simulate
+from timemachine.potentials import bonded
+
+# test that we do not invert stereochemical barriers at the end-states for various susceptible transformations.
+# most of these tests proceed by measuring the chiral volume defined by 4 atoms and ensuring that they're consistent
+# at both end-states.
+
+
+def minimize_bfgs(x0, U_fn):
+    N = x0.shape[0]
+
+    def u_bfgs(x_flat):
+        x_full = x_flat.reshape(N, 3)
+        return U_fn(x_full)
+
+    grad_bfgs_fn = jax.grad(u_bfgs)
+    res = scipy.optimize.minimize(u_bfgs, x0.reshape(-1), jac=grad_bfgs_fn)
```

Comment:
nit: would it make sense to specify `method="L-BFGS-B"`, or update the name of the function to just something like `minimize_potential`?

Reply:
i think scipy.minimize has become synonymous with BFGS at this point, but I agree that we should be consistent. I will rename to minimize_scipy just incase something better than BFGS comes along (I hope so!)